src/backend/aspen/workflows/pangolin/run_mass_update.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# TODO: fix pipefail flags to be informative
+#set -Eex pipefail
+
+# activate miniconda
+eval "$($HOME/miniconda/bin/conda shell.bash hook)"
+conda init
+conda activate pangolin
+
+# record pangolin version:
+PANGOLIN_VERSION=$(pangolin -pv)
```

Comment:
Any idea what's up with the scorpio versions? I think useher/pangolearn versions match pangolin versions, but this is odd:

```
SCORPIO_v0.1.10
```

Reply:
No, I was not aware this was a possible value until now. That doesn't match the pangolin version so to support this we would have to extract a version string somewhere :/ @danrlu do you know what this is?